src/platform/nm-linux-platform.c

```diff
@@ -7094,6 +7096,13 @@ constructed (GObject *_object)
 	                                (EVENT_CONDITIONS | ERROR_CONDITIONS | DISCONNECT_CONDITIONS),
 	                                 event_handler, platform);
 
+	priv->genlh = nl_socket_alloc ();
```

Comment:
This is missing a matching `nl_socket_free` on `finalize`. Will fix when rebasing.